driver/razerkbd_driver.c

```diff
@@ -867,6 +868,7 @@ static ssize_t razer_attr_read_device_type(struct device *dev, struct device_att
         break;
 
     case USB_DEVICE_ID_RAZER_HUNTSMAN_MINI:
+    case USB_DEVICE_ID_RAZER_HUNTSMAN_MINI_JP:
         device_type = "Razer Huntsman Mini\n";
```

Comment:
Maybe it would be good to call the device also "Razer Huntsman Mini (JP)" here? Although probably the marketing name for this just "Razer Huntsman Mini", right?